src/common/components/configurationForm/index.jsx

```diff
@@ -203,7 +203,7 @@ const ConfigurationForm = () => {
             autoCorrect="off"
             autoCapitalize="none"
             min={20}
-            max={9999}
+            max={Number.MAX_SAFE_INTEGER}
```

Comment:
999 connections even dont make sense, so I dont see reason for this

Reply:
Suppose I have a 4 socket EPYC server, with 768 cores and 2TB of ram.... At 15MB per connection = ~83,200 theoretical max connections.

I'm mostly concerned with memory, however, I don't see a logical reason to enforce a maximum on any of these values. (How did we land on 9999?)